Update packaging for 0.10

scollazo opened the pull request:
Use gunicorn instead of wsgi

scollazo committed:
Update packaging and use gunicorn instead of wsgi

scollazo committed:
Remove wsgi references

jhsimpson commented:
This PR is failing checks because of the pip/setuptools bug in the ubuntu 14.04 packaged version of pip 1.5.4-1.  

http://jenkins-ci.archivematica.org/job/artefactual/job/archivematica-storage-service/74/console

It might be necessary to specify the version of pip and setuptools required in the storage service requirements.yml file, in order to get around this?  There is a fix in the ansible deployment script https://github.com/artefactual-labs/ansible-archivematica-src/pull/103 perhaps it is better to fix it here, in the requirements file?

jhsimpson committed:
fixing versions of python dependencies
to guarantee reproducible deployments.

In particular, adding pip and setuptools
to work around python-pip bug in ubuntu 14.04

jhsimpson commented:
this is :+1:  from me, I tested and as it stands this is working for me.
